multicluster/Manifest/gtp-crd.yaml

```diff
@@ -36,106 +16,134 @@ spec:
-                        - RTT
-                        - ROUNDROBIN
-                        - STATICPROXIMITY
-                        description: "The traffic distribution policy supported in multi-cluster deployment"
-                      secLbMethod:
-                        type: string
-                        description: "The traffic distribution policy supported among clusters under a group in local-first, canary or failover"
-                      targets:
-                        items:
-                          required: ["destination"]
-                          properties:
-                            destination:
-                              type: string
-                              description: "Ingress or LoadBalancer service endpoint in each cluster. Should match with name of GSE"
-                            weight:
-                              type: integer
-                              minimum: 1
-                              maximum: 100
-                              description: "Proportion of traffic to be maintained across clusters. For canary proportion is percentage"
-                            rule:
-                              type: string
-                            CIDR:
-                              type: string
-                              description: "CIDR to be used in local-first to determine the scope of locality"
-                            primary:
-                              type: boolean
-                              description: "Is this destination a primary cluster or a backup cluster in failover deployment. Possible values: True or False"
-                          type: object
-                        type: array
-                    type: object
-                  monitor:
-                    items:
+  versions: 
+  - name: v1beta1
+    served: true
+    storage: true
+    subresources:
+      status: {}
+    additionalPrinterColumns:
+      - name: Status
+        type: string
+        description: "Current Status of the CRD"
+        jsonPath: .status.state
+      - name: Message
+        type: string
+        description: "Status Message"
+        jsonPath: .status.status_message 
+    schema:
+     # openAPIV3Schema is the schema for validating custom objects.
+      openAPIV3Schema:
+        type: object
+        required: ["apiVersion","kind","metadata","spec"]
+        properties:
+          apiVersion:
+            type: string
+          kind:
+            type: string
+          metadata:
+            #required: ["name","namespace"]
+            properties:
+              name:
+                type: string
+              #namespace:
+              #  type: string
+            type: object
+          status:
+            type: object
+            properties:
+              state:
+                 type: string
+              status_message:
+                 type: string
+          spec:
+            required: ["serviceType","hosts","status"]
```

Comment:
status is not required field here, please take the correct spec for this CRD from viswa.